geodiff/src/drivers/sqlitedriver.cpp

```diff
@@ -678,7 +687,7 @@ static std::string sqlForUpdate( const std::string &tableName, const TableSchema
     {
       // compare date/time values using datetime() because they may have
       // multiple equivalent string representations (see #143)
-      sql += sqlitePrintf( " ( ?%d = 0 OR datetime(\"%w\") IS datetime(?%d) ) ", i * 3 + 2, tbl.columns[i].name.c_str(), i * 3 + 1 );
+      sql += sqlitePrintf( " ( ?%d = 0 OR STRFTIME('%%Y-%%m-%%d %%H:%%M:%%S.%%fZ', \"%w\") = STRFTIME('%%Y-%%m-%%d %%H:%%M:%%S.%%fZ', ?%d) ) ", i * 3 + 2, tbl.columns[i].name.c_str(), i * 3 + 1 );
```

Comment:
similar to above + please update the comment that datetime() skips fractional seconds, so we use strftime() explicitly

Reply:
Fixed as well